tests/ChannelJsonImportExportTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FriendsOfSylius\SyliusImportExportPlugin;
+
+final class ChannelJsonImportExportTest extends AbstractJsonImportExportTest
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->loadJsonFixtures('currency', <<<LOL
+[
+    {
+        "Code": "USD"
+    },
+    {
+        "Code": "PLN"
+    }
+]
+LOL
+        );
+        $this->loadJsonFixtures('locale', <<<LOL
+[
+    {
+        "Code": "en_US"
+    },
+    {
+        "Code": "pl_PL"
+    }
+]
+LOL
+        );
+        $this->loadJsonFixtures('zone', <<<LOL
+[
+    {
+        "Code": "NA",
+        "Name": "North America",
+        "Type": "country",
+        "Scope": "all",
+        "Members": ["US", "CA"]
+    }
+]
+LOL
+        );
+    }
+
+    protected function provideName(): string
+    {
+        return 'channel';
+    }
+
+    protected function provideJsonData(): string
+    {
+        return <<<LOL
+[
+    {
+        "Code": "US_WEB",
+        "Name": "US Web Store",
+        "Description": "sample_description",
+        "Hostname": "localhost",
+        "Color": "GreenYellow",
+        "Enabled": true,
```

Comment:
Could we add a second test channel here with all the boolean negated (so that we are sure they are imported and not default values)?